ors-engine/src/main/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopper.java

```diff
@@ -414,8 +414,7 @@ protected void postProcessing(boolean closeEarly) {
                 calculateContours();
                 List<Profile> profiles = fastIsochroneFactory.getFastIsochroneProfiles();
                 for (Profile profile : profiles) {
-                    Weighting weighting = ((ORSWeightingFactory) createWeightingFactory()).createIsochroneWeighting(profile, new PMap(profile.getName()).putObject("isochroneWeighting", "true"));
-
+                    Weighting weighting = ((ORSWeightingFactory) createWeightingFactory()).createIsochroneWeighting(profile);
```

Comment:
I wonder whether there was a reason to have a seperate `IsochroneWeighting`?

Reply:
Do you mean the removed PMap entry `"isochroneWeighting"`? I couldn't find any use of it, so looks like a historic thing which became obsolete t some point...